CD 중 서버 경로 에러 해결

### 🐞설명

CD 중 서버 체크 에러를 해결해야 합니다.

### ⛰️ 맥락

1. 병합을 한다.
2. CI 통과
3. CD 중 서버 체크 시 green이 없다고 에러 발생

### 🌏 환경

ubuntu-latest

### 📄 로그

```shell
Run appleboy/ssh-action@master
  with:
    username: ubuntu
    host: ***
    key: ***
    script_stop: true
    script: sudo docker stop green
  sudo docker rm green
    port: 22
    protocol: tcp
    timeout: 30s
    command_timeout: 10m
    proxy_port: 22
    proxy_protocol: tcp
    proxy_timeout: 30s
  env:
    CURRENT_UPSTREAM: green
    CURRENT_PORT: 8081
    STOPPED_PORT: 8080
    TARGET_UPSTREAM: blue
Run echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
  echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    CURRENT_UPSTREAM: green
    CURRENT_PORT: 8081
    STOPPED_PORT: 8080
    TARGET_UPSTREAM: blue
    GITHUB_ACTION_PATH: /home/runner/work/_actions/appleboy/ssh-action/master
Run entrypoint.sh
  entrypoint.sh
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    CURRENT_UPSTREAM: green
    CURRENT_PORT: 8081
    STOPPED_PORT: 8080
    TARGET_UPSTREAM: blue
    GITHUB_ACTION_PATH: /home/runner/work/_actions/appleboy/ssh-action/master
    INPUT_HOST: ***
    INPUT_PORT: 22
    INPUT_PROTOCOL: tcp
    INPUT_USERNAME: ubuntu
    INPUT_PASSWORD: 
    INPUT_PASSPHRASE: 
    INPUT_KEY: ***
    INPUT_KEY_PATH: 
    INPUT_FINGERPRINT: 
    INPUT_PROXY_HOST: 
    INPUT_PROXY_PORT: 22
    INPUT_PROXY_USERNAME: 
    INPUT_PROXY_PASSWORD: 
    INPUT_PROXY_PASSPHRASE: 
    INPUT_PROXY_KEY: 
    INPUT_PROXY_KEY_PATH: 
    INPUT_PROXY_FINGERPRINT: 
    INPUT_TIMEOUT: 30s
    INPUT_PROXY_TIMEOUT: 30s
    INPUT_COMMAND_TIMEOUT: 10m
    INPUT_SCRIPT: sudo docker stop green
  sudo docker rm green
    INPUT_SCRIPT_STOP: true
    INPUT_ENVS: 
    INPUT_ENVS_FORMAT: 
    INPUT_DEBUG: 
    INPUT_ALL_ENVS: 
    INPUT_REQUEST_PTY: 
    INPUT_USE_INSECURE_CIPHER: 
    INPUT_CIPHER: 
    INPUT_PROXY_USE_INSECURE_CIPHER: 
    INPUT_PROXY_CIPHER: 
    INPUT_SYNC: 
Will download drone-ssh-1.7.7-linux-amd64 from https://github.com/appleboy/drone-ssh/releases/download/v1.7.7
======= CLI Version =======
Drone SSH version 1.7.7
===========================
err: Error response from daemon: No such container: green
2024/11/14 01:41:35 Process exited with status 1
Error: Process completed with exit code 1.
```


### 🙋🏻 덧붙일 말

_No response_

--- a/.github/workflows/cicd.yml
+++ b/.github/workflows/cicd.yml
@@ -59,10 +59,10 @@ jobs:
     steps:
       - name: Set target IP
         run: |
-          STATUS=$(curl -o /dev/null -w "%{http_code}" "http://${{ secrets.LIVE_SERVER_IP }}/env")
+          STATUS=$(curl -o /dev/null -w "%{http_code}" "https://${{ secrets.LIVE_SERVER_DOMAIN }}/env")
           echo $STATUS
           if [ $STATUS = 200 ]; then
-            CURRENT_UPSTREAM=$(curl -s "http://${{ secrets.LIVE_SERVER_IP }}/env")
+            CURRENT_UPSTREAM=$(curl -s "https://${{ secrets.LIVE_SERVER_DOMAIN }}/env")
           else
             CURRENT_UPSTREAM=green
           fi
@@ -91,7 +91,7 @@ jobs:
       - name: Check deploy server URL
         uses: jtalk/url-health-check-action@v3
         with:
-          url: http://${{ secrets.LIVE_SERVER_IP }}:${{env.STOPPED_PORT}}/env
+          url: https://${{ secrets.LIVE_SERVER_IP }}:${{env.STOPPED_PORT}}/env
           max-attempts: 4
           retry-delay: 10s
 

